### DISCUSSION TITLE
导出/入软件配置

### DISCUSSION THREAD

**Discussion by happytbs** [Class-Widgets/Class-Widgets#300] [Feature]
### 检查清单

- [X] 我已更新到最新版 ![最新的版本号](https://img.shields.io/github/v/release/RinLit-233-shiroko/Class-Widgets?include_releases&style=flat-square&label=) ，并看过[最新提交](https://github.com/RinLit-233-shiroko/Class-Widgets/commits/main/)，确认我想要的功能还没有实现。
- [X] 我已在 [Issues](https://github.com/RinLit-233-shiroko/Class-Widgets/issues?q=label%3ABug) 中检索，确认这一 Bug 未被提交过。
- [X] 我认为我想要的功能是用户广泛需要的，插件不能替代的，且没有超出 **软件本来** 的开发目标，而非 **添加与课表及教学辅助无关的内容** 。
- [X] 我已知晓由于开发者时间精力有限，我提交的功能请求可能需要比较久的时间才能实现。

### 背景与动机

![image](https://github.com/user-attachments/assets/9b5d195f-40b0-4d5c-8516-8922213214ae)
更新软件之后导出导入课表的同时软件设置和倒计时等自定义功能还需要手动去切换

### 想要实现或优化的功能

希望在配置文件界面提供导出/入软件配置

### 最后一步

- [X] 我认为上述的描述已经足以详细，以便开发人员理解您想要的功能。如果我的 issue 没有按照上述的要求填写，可能会被无条件关闭。

### 不要乱下载别人发的文件，特别是名字是fix.zip的文件。您是否遇见过这样的文件？

否

**Comment by RinLit-233-shiroko** [comment]
好的

**Comment by Detritalw** [comment]
可以直接拷贝`config`这个文件夹的文件，包括所有配置和课表
